.github/workflows/ci_benchmark.yml

```diff
@@ -87,6 +87,11 @@ jobs:
               | sed "/Traceback \|failed$\|PERFORMANCE DECREASED/ s/^/::error::/" \
               | tee benchmarks.log
 
+          # Error out the action
+          if grep -q "Traceback \|failed\|PERFORMANCE DECREASED" benchmarks.log ; then
+              exit 1
```

Comment:
If it fails here, does it prevent the upload step? We want to still upload right, especially when things are failing and need attention?

Reply:
Nice catch! Thanks!